src/keymap.cpp

```diff
@@ -148,6 +148,13 @@ static const std::vector<OpDesc> opdescs = {
 		_("Open URL of article, feed, or entry in URL view"),
 		KM_FEEDLIST | KM_ARTICLELIST | KM_ARTICLE | KM_URLVIEW
 	},
+	{
+		OP_OPENINBROWSER_NONINTERACTIVE,
+		"open-in-browser-noninteractively",
+		"",
+		_("Open URL of article, feed, or entry in non-interactive browser"),
```

Comment:
I think this should've reflected the change too:

```suggestion
		_("Open URL of article, feed, or entry in a browser, non-interactively"),
```

Reply:
Done: https://github.com/newsboat/newsboat/compare/80f1766b060b0b2d6d7cb64737cd4ca01f4d2270..7f6ce535a715779e5ff937e6ff56bda4c7831c18